cnxeasybake/oven.py

```diff
@@ -1334,3 +1340,15 @@ def copy_w_id_suffix(elem, suffix="_copy"):
     for id_elem in mycopy.xpath('//*[@id]'):
         id_elem.set('id', id_elem.get('id') + suffix)
     return mycopy
+
+
+def fetch_nodes(url):
+    """Get the contents at url, return parsed nodes child of body tag."""
+    req = requests.get(url)
+    if req.ok:
+        html = etree.HTML(req.text)
+        nodes = html.xpath('body/*')
```

Comment:
Could fetch handle `#id123` in the URL (and error/warn if there was a problem)? I think that feature would add support for real transclusion.


💭  how much extra work would it be to have `fetch(url)` take an optional selector as an additional parameter? I'm not suggesting that it gets added yet, just wondering how much work it would be.